main/src/main/scala/sbt/internal/Load.scala

```diff
@@ -1462,7 +1462,7 @@ private[sbt] object Load {
 
   /**
    * Creates a classloader with a hierarchical structure, where the parent
-   * classloads the dependency classpath and the return classloader classloads
+   * class loads the dependency classpath and the return classloader class loads
```

Comment:
IMO _classload_ as a verb is fine

Reply:
I spoke w/ one of my coworkers who works in Scala about this before submitting the PR, the support for this change was a few lines later which uses `class load` in the `@return` statement:
https://github.com/sbt/sbt/blob/0a1a36a2b34a5eefdb016b6eae6dd4f8ea43b5ca/main/src/main/scala/sbt/internal/Load.scala#L1464-L1471

If code is written in a way that shows it believes a word is a word, I don't try to rewrite things based on my personal beliefs, I really try to follow Rome, and this is where I found Rome.